src/api/api.js

```diff
@@ -1153,11 +1153,11 @@ export {
   getUserModelInfoApi,
   getBookByIdApi,
   getBookRecommendListApi,
-  getCategoriesApi,
+  getBookCategoriesApi,
   getAllBookApi,
-  addBookRentApi,
-  updateRentBookApi,
-  getUserSeatInfoApi,
+  addBookRentReservationApi,
+  updateBookRentReservationApi,
+  getUserSeatReserInfoApi,
```

Comment:
There is a **getUserBookReservationApi** below. I just wondering it would be better if these two apis are in a same format, like **getUserSeatReserInfoApi**, **getUserBookReserInfoApi**.

And it seems than "Rent" and "Reservation" are the same meaning here, so maybe we can use just one of them? I mean, all **Rent** or all **Reservation**.

Reply:
PS: this is a "updateRecommendAmountApi" should be "updateRecommendAmountByIdApi".

In my opinion, whether to add "ById" to the function name depends on whether to add the corresponding unique identifier Id when calling this parameter. If necessary, it is best to add it. So everyone will undertand this function means get something from **this** type of data, like updating sth from this book, getting book list from this category. Just my opinion.

Just keep it consistent. If we added ById, just add the same type. If you agree to add it, so just add it. If you thought we don’t need it, just don’t add it, and remove others.